### DISCUSSION TITLE
v38 broke my special workspace config

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5379] [Q&A] [answered]
workspace=special,gapsin:20,gapsout:120,bordersize:2,border:true

now seems to do nothing, and i'm not seeing anything updated in the documentation

**Comment by vaxerski** [comment, marked as answer]
try using it as a named special workspace (e.g. `special:one`)

**Reply by Kommynct** [reply]
Followup question, how do I set a windowrule to go to a special named workspace, i now have it set as scratchpad

windowrulev2 = bordersize 2, workspace:scratchpad, class:(firefox)

edit: yes, I tested it, i'm dumb sorry for asking

**Reply by vaxerski** [reply]
workspace: is a selector. You want `windowrulev2 = workspace special:one, ...`

**Reply by Azarattum** [reply]
Using named special workspace breaks taskbar module in waybar. Clicking on a window in special named workspace always brings the default/empty/unnamed one. The problem was fixed by using an unnamed special workspace, how do I fix it now?

**Reply by Azarattum** [reply]
using `name:special` seem to have solved my use case
